front/app/components/ProjectAndFolderCards/ProjectAndFolderCardsInner.tsx

```diff
@@ -53,23 +54,20 @@ const ProjectAndFolderCardsInner = ({
   statusCounts,
   showTitle,
   showSearch,
-  search,
```

Comment:
This was from a previous solution I tried, forgot to remove it from the props here. It wasn't being passed in so the `rootLevelOnly` check below was only resolving to true